### PR TITLE
Issue #139 - Landing page layout

### PR DESCRIPTION
**Pull request by krozett**
This should finish up the landing page layout

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -5,4 +5,42 @@ class Organization < ActiveRecord::Base
   has_many :positions
   has_many :users, through: :positions
 
+  def engagement_points(type = :all)
```
Comment (ebollens) [inline app/models/organization.rb]: I don't understand where these values are getting set/incremented.

Comment (krozett) [inline app/models/organization.rb]: Somehow I forgot to add the actual point setting. It's all in there now.

File: app/controllers/groups_controller.rb [hunk 2]
```diff
@@ -40,7 +43,10 @@ def show
   end
 
   def destroy
+ap "DEP!"
```
Comment (ebollens) [inline app/controllers/groups_controller.rb]: What is this?

Comment (krozett) [inline app/controllers/groups_controller.rb]: Whoops, debug. BTW `ap` is a great debugging tool if you haven't used it yet.

**Pull request merged**